docs-source/content/faq/configmaps.md

```diff
@@ -0,0 +1,33 @@
+> I need to provide an instance with access to a Config Map
+
+Configuration information in Kubernetes may be provided by a 
+[ConfigMap](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#create-a-configmap), 
+which consists of a set of key-value pairs. Each entry may be accessed by one or more operator-managed nodes
+as a read-only text file. Access can be provided across the domain, within a single cluster, or for a single server.
+In each case, the access is configured within the ``ServerPod`` element of the configuration. For example, given
```

Comment:
"In each case, the access is configured within the ServerPod element of the configuration."

looks incorrect. (1) Element name is 'serverPod' not 'ServerPod'., (2) The cluster configuration is in 'clusters', not 'ServerPod', as per your example below.

How about "Access is configured within a domain resource using its 'serverPod', 'adminServer', 'managedServers', or 'clusters' elements. (I'm not sure if all 4 have access - please confirm.)

Also, please turn 'domain resource' into a link that references the "Domain resource" doc page.